test/lib/clients.ts

```diff
@@ -0,0 +1,78 @@
+import jwt from 'jsonwebtoken'
+import { GoTrueApi, GoTrueClient } from '../../src/index'
+
+export const AUTO_CONFIRM_ENABLED_GOTRUE_PORT = 9999
+export const SIGNUP_ENABLED_AUTO_CONFIRM_ENABLED_PORT = 9998
+export const AUTO_CONFIRM_DISABLED_GOTRUE_PORT = 9997
+
+export const GOTRUE_URL_AUTO_CONFIRM_ENABLED = `http://localhost:${AUTO_CONFIRM_ENABLED_GOTRUE_PORT}`
+export const GOTRUE_URL_SIGNUP_ENABLED_AUTO_CONFIRM_ENABLED = `http://localhost:${SIGNUP_ENABLED_AUTO_CONFIRM_ENABLED_PORT}`
+export const GOTRUE_URL_AUTO_CONFIRM_DISABLED = `http://localhost:${AUTO_CONFIRM_DISABLED_GOTRUE_PORT}`
```

Comment:
would it be better to throw all of these into a `.test.env` file?

Reply:
The test suite and package doesn't rely on dotenv or have any envier config.

I'd still have to check in a .env file with these settings and then inline them like

```
export const AUTO_CONFIRM_ENABLED_GOTRUE_PORT = process.env. AUTO_CONFIRM_ENABLED_GOTRUE_PORT
```

and they *must* match the docker compose settings, which are not enviers.

So, therefore, keeping values in the client config here.